quda_interface.h

```diff
@@ -86,6 +86,33 @@
 #include "su3.h"
 #include "solver/solver_params.h"
 
+typedef struct Quda_mg_params_t{
```

Comment:
These should be in another file named ```quda_types.h```, for example, such that they can be included in ```read_input.l``` even when QUDA is not being used.